projects/valtimo/components/src/lib/components/multiselect-dropdown/multiselect-dropdown.module.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {MultiselectDropdownComponent} from './multiselect-dropdown.component';
+import {NgMultiSelectDropDownModule} from 'ng-multiselect-dropdown';
+
+@NgModule({
+  declarations: [MultiselectDropdownComponent],
+  imports: [CommonModule, ReactiveFormsModule, NgMultiSelectDropDownModule.forRoot(), FormsModule],
```

Comment:
Why .forRoot() is this module is presumably imported multiple times if imported in multiple places?

Reply:
According to the website of the plugin the forRoot() was needed

Reply:
I'll test it without the .forRoot()

Reply:
In the example this was done in the AppModule please do test if it works without forRoot() here